Snowflake.Data/Configuration/EasyLoggingConfigParser.cs

```diff
@@ -80,5 +98,29 @@ private void CheckForUnknownFields(string fileContent)
                 .ToList()
                 .ForEach(unknownKey => s_logger.Warn($"Unknown field from config: {unknownKey.Name}"));
         }
+
+        private void CheckIfValidPermissions(FileStream fileStream, string filePath)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return;
+            }
+
+#if NET8_0_OR_GREATER
+            var unixFileMode = File.GetUnixFileMode(fileStream.SafeFileHandle);
```

Comment:
Can we exclude Mono.Unix from dependencies for .NET 8? @sfc-gh-knozderko wdyt?

Reply:
I tried but there are many other places that uses de Mono.Unix. If this change is needed I prefer to do it in a a new task instead
![Screenshot 2024-05-16 at 10 49 05 AM](https://github.com/snowflakedb/snowflake-connector-net/assets/130484280/9c18dcff-8feb-4ed6-b8b1-2587dbdc0c5c)